### PR TITLE
CI: use only medium runners on Openstack

### PR DESCRIPTION
**Pull request by jrusz**
Medium runners might be enough and will give us more jobs in parallel

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -335,12 +336,12 @@ New OSTree:
...
     matrix:
       - RUNNER:
-          - rhos-01/fedora-35-x86_64-large
-          # - rhos-01/fedora-36-x86_64-large
-          - rhos-01/rhel-8.7-nightly-x86_64-large
-          - rhos-01/rhel-9.1-nightly-x86_64-large
-          - rhos-01/centos-stream-8-x86_64-large
-          - rhos-01/centos-stream-9-x86_64-large
+          - rhos-01/fedora-35-x86_64
+          # - rhos-01/fedora-36-x86_64
```
Comment (atodorov) [inline .gitlab-ci.yml:340]: ATM there are fedora-36-x86_64 and fedora-36-x86_64-large runner definitions so you can uncomment this line.

**Review by atodorov — CHANGES_REQUESTED** [review]
Minor change requested.

**Review by atodorov — DISMISSED** [review]

**Comment by jrusz** [comment]
Hm, this will probably require some reshuffling in the `ostree-rebase.sh` or somehow forcing `virt-install` to use less memory, he doesn't listen to my `--ram 1536` we can't spawn 2 VMs with 3 GB memory on a 4 GB machine.

**Comment by jrusz** [comment]
I've decided to split the `ostree-rebase.sh` test into bios and uefi scenario. This won't add load on the CI as 2 medium runners = 1 large runner and after this change we're still getting 26 more Openstack runners to use (was supposed to be 30) which is still good IMO.

**Review by ondrejbudai — APPROVED** [review]
Thanks!

**Pull request merged**